.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -61,7 +62,16 @@ assignees: ''
 **What did you expect to happen?**
 
 
-**What actually happened? Please include the actual, raw output from ESLint.**
+**What actually happened? Please copy-paste the actual, raw output from ESLint.**
 
 
+**Steps to reproduce this issue:**
+
+<!-- Please tell us exactly how to see the issue you're describing -->
+
+1. 
+1. 
+1. 
```

Comment:
This is sometimes useful, but I think far more often redundant with other fields. Can we instead ask for the exact steps only when we are unable to reproduce the issue?

Reply:
I’d rather a bit of redundancy upfront. When I’m looking at bugs, too often people leave out repro steps and not enough people ask. Let’s just get it out of the way early.

Reply:
I'm not seeing many issues where additional info is needed, aside from those where people don't fill out the existing sections or completely ignore the template.

What would be repro steps for a bug report related to a rule?

Reply:
1. Type this code into editor “”
2. Configure rule with these options
3. Run ESLint
4. See unexpected rule result

Can you explain your concern over adding this? To me, I’d just like KOR info upfront as I don’t feel like we always get the best bug reports. Especially if we are going to start letting non-team members so triage, I think having this info will really help.

Reply:
> Can you explain your concern over adding this?

Only that a template with many fields might be discouraging for users who'd like to report a bug.

Reply:
I can understand that. My feeling is that this additional ask isn’t enough to dissuade people from filing an issue.

can we try this and see what happens? We can always remove it if it’s a problem, but I think this is important to include to help out the triage team, who may not be as familiar with ESLint internals.

Reply:
Agreed!